src/components/EditSections/EditUserInfo/components/EditUserProfilePicture/EditUserProfilePicture.js

```diff
@@ -100,7 +102,16 @@ const EditUserProfilePicture = ({ profilePictureId, form, personal }) => {
   }, []);
 
   const onFileChange = async (e) => {
-    if (e.target.files && e.target.files.length > 0) {
+    const maxFileSizeInBytes = profilePictureMaxFileSize * 1024 * 1024;
+    if (maxFileSizeInBytes && e.target.files[0].size > maxFileSizeInBytes) {
+      calloutRef.current.sendCallout({
+        type: 'error',
+        message: `Photo size exceeds the ${profilePictureMaxFileSize}MB limit.
+        Please choose a photo with a size of ${profilePictureMaxFileSize}MB  or less.`,
+      });
+      // eslint-disable-next-line no-console
+      console.warn('max file size can be ' + profilePictureMaxFileSize + 'mb.');
+    } else if (maxFileSizeInBytes && e.target.files && e.target.files.length > 0) {
```

Comment:
Please check if you have to adjust the order of check.
1. what if maxFileSize is 0 for some odd reason? 
2. check if e.target.files.length > 0 and also maxFileSize, then comes the check for file size greater than maxFileSize.
3. Error message needs to be i18'zed